src/arch/bsdsocketinterf.cpp

```diff
@@ -17,6 +17,14 @@
 #include "devlog.h"
 #include <string.h>
 
+#ifdef __ZEPHYR__
```

Comment:
The design of 4diac FORTE's bsdsocketinerface is such that these kind of includes and configurations should be done in the os specific includes and not here. 

Therefore please move this to `sockhand.h`in the zephyr directory. A good example is the `sockhand.h`of freeRTOS.

Reply:
Done.